admin/services/data-access/organisation-bulk-upload.data.service.js

```diff
@@ -0,0 +1,70 @@
+const sqlService = require('./sql.service')
+const uuid = require('uuid')
+const R = require('ramda')
+
+const organisationBulkUploadDataService = {
+  /**
+   * Determine if an existing organisation upload job is running.
+   * @returns {Promise<boolean>}
+   */
+  isExistingJob: async function isExistingJob () {
+    const sql = `SELECT *
```

Comment:
would it be slightly more optimal to do a count here?

Reply:
Good point!  Now changed.